src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -720,8 +720,13 @@ export class PatientViewPageStore {
 
         for (const mutation of this.uncalledMutationData.result) {
             mutationId = PatientViewPageStore.getMutationId(mutation);
-            idToMutations[mutationId] = idToMutations[mutationId] || [];
-            idToMutations[mutationId].push(mutation);
+            let mutationList:Mutation[]|undefined = idToMutations[mutationId];
+            // only add uncalled mutation if it is called in another sample of
+            // this patient
+            if (mutationList !== undefined) {

```

Comment:
I think you could get rid of mutationList and just just write if (mutationId in idToMutations) i.e. use "in" operator